tox.ini

```diff
@@ -14,6 +14,14 @@ deps=
 	-clegacy.constraints
 	-rtest-requirements.in
 
+[testenv:py38]
```

Comment:
I guess you've added this for debugging the CI failure relating to rpm-py-installer.  Could you drop it now please?  That problem has disappeared.